handbook/people-ops/onboarding/general_onboarding.md

```diff
@@ -54,7 +59,8 @@
 ### Communication and documentation
 
 - We're a handbook-first company. Learn all about the handbook on its [handbook page](../../usage.md).
-- Join relevant Slack channels. The main channels are [listed in the handbook](../../communication/team_chat.md#channels), but it's a good idea to look through all the channels in our Slack and join all that seem relevant!
+- We use Slack for team chat. Please read the [Slack handbook page](../../communication/team_chat.md) to understand what we use Slack for and how we use it. 
```

Comment:
@InesRoitman Since I was here already, I noticed we link to Slack channels but not the overall Slack guidelines here. I added a bullet to read the full page since it's super important 😁